### DISCUSSION TITLE
做wordlist和concordancer的generate figure时，中文无法正常显示。

### DISCUSSION THREAD

**Discussion by vraiment160** [BLKSerene/Wordless#32] [Q&A] [answered]
如图。请问我是哪里没设置好呢？谢谢
<img width="1675" alt="Snipaste_2023-12-20_19-43-04" src="https://github.com/BLKSerene/Wordless/assets/35804875/d83840c8-9ba1-4a46-9ec0-8bf8ffc6a6ad">

**Comment by BLKSerene** [comment, marked as answer]
菜单 - Preferences - Settings - Figures - Line charts 里选一个支持显示中文的字体 多换几个试试效果
词云图和网络图 也是这里（Figures - Word Clouds / Network Graphs）进行设置 改完记得保存 然后重新生成图表

**Reply by vraiment160** [reply]
谢谢！完美解决。我选择了songti sc。仅供后来者参考。
